lib/qless/server/static/css/style.css

```diff
@@ -134,6 +134,20 @@ body { padding:0; margin:0; }
 .subnav li.current a { background:#fff; font-weight:bold; color:#ce1212;}
 .subnav li a:active { background:#b00909;}
 
+.subnav.subnav-fixed.page-layout {
+  position: unset;
+	height: 100%;
+	margin-top: 20px;
+	background-image: -webkit-gradient(linear, left top, left bottom, color-stop(0%,#f5f5f5), color-stop(100%,#F5F5F5));
+	border: 10px solid #F2DEDE;
+	border-radius: 4px;
+}
+
+ul.nav.nav-pills.rows {
+  display: flex;
+  flex-flow: column wrap;
```

Comment:
I would consider using the broken up rules for clarity. Makes it easier for readability, but that's my opinion.

```
flex-direction: column;
flex-wrap: wrap;
```